util/src/main/scala/org/ensime/util/file.scala

```diff
@@ -115,6 +115,9 @@ package object file {
         case t: Throwable => file.getAbsoluteFile
       }
 
+    def descendsFrom(parent: File): Boolean =
+      file.getParent.startsWith(parent.getAbsolutePath)
```

Comment:
Thanks for the guidance @fommil, I'm not familiar with `vfs2` yet. `isAncestor` or `isDescenant` look to be what I'll need.